app/uk/gov/hmrc/bindingtariffclassification/search/Search.scala

```diff
@@ -101,6 +102,7 @@ object Filter {
   private val minDecisionEndKey = "min_decision_end"
   private val commodityCodeKey = "commodity_code"
   private val goodDescriptionKey = "good_description"
+  private val keywordsKey = "keywords"
```

Comment:
I think convention is to do query params Singular. I may be wrong

Reply:
Ok